src/Esia/Signer/SignerPKCS7.php

```diff
@@ -20,28 +20,28 @@ class SignerPKCS7 implements SignerInterface
      *
      * @var string
      */
-    private $certPath;
+    protected $certPath;
```

Comment:
Keep properties as private

Reply:
Did you mean that CliSignerPKCS7 should have private `$certPath` and another?
It looks like copy & past